database/src/migrations/release.0.31/biohub.sql

```diff
@@ -2320,6 +2411,7 @@ CREATE TABLE survey_summary_detail(
     study_area_id                   varchar(100)      NOT NULL,
     parameter                       varchar(100)      NOT NULL,
     stratum                         varchar(100)      NOT NULL,
+    parameter_method                varchar(200),
     parameter_value                 numeric(10, 0),
     parameter_esitmate              numeric(10, 0),
```

Comment:
Small spelling mistake here, and in the associated comment.

Reply:
pushing a fix